packages/core/contracts/financial-templates/common/FundingRateApplier.sol

```diff
@@ -34,10 +34,10 @@ contract FundingRateApplier is Testable, Lockable {
     FinderInterface public fpFinder;
 
     // Last time the `cumulativeFundingRateMultiplier` was updated.
-    uint256 lastUpdateTime;
+    uint256 public lastUpdateTime;
 
     // Identifier in funding rate store to query for.
-    bytes32 identifier;
+    bytes32 public priceIdentifier;
```

Comment:
`priceIdentifier` is more consistent with how we previously defined this variable in the EMP.

Reply:
I think we should rename this to fundingRateIdentifier, #2086 , but can do in another PR

Reply:
have implemented this.